browser/components/Shared/Button.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
```

Comment:
We're only exporting the StyledComponent here.
Here we should create and export a component, that will accept a props object with an onClick function and attach it to the button's onClick event.